src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx

```diff
@@ -29,6 +29,69 @@ Before installing the integration, make sure that you meet the following require
+
+<Callout variant="tip">
+
+The integration will use Zookeeper only for discovering brokers and will not retrieve metrics from it.
+</Callout>
+
+### Topic listing [#topic-listing]
+
+To correctly list the topics processed by the brokers, the integration needs to to contact brokers over the Kafka protocol. Depending on how the brokers are configured, this might require setting up SSL and/or SASL to match the broker configuration.
+
+### Broker monitoring (JMX) [#broker-monitoring]
+
+The Kafka integration queries JMX, a standard Java extension for exchanging metrics in Java applications. JMX is not enabled by default in Kafka brokers, and you need to enable it for metrics collection to work properly. JMX requires RMI to be enabled, and the RMI port needs to be set to the same port as JMX.
+
+You can configure JMX to use username/password authentication, as well as SSL. If such features have been enabled in the broker's JMX settings, you need to configure the integration accordingly.
+
+<Callout variant="important">
+
+We do not recommend enabling anonymous and/or unencrypted JMX/RMI access on public or untrusted network segments because this poses a big security risk.
+</Callout>
+
+### Producer/consumer monitoring (JMX) [#producer]
+
+Producers and consumers written in Java can also be monitored through the same mechanism (JMX). JMX needs to be enabled and configured on those applications where it is not enabled by default.
+
+Non-Java producers and consumers do not support JMX and are therefore not supported by the Kafka integration.
+
+### Connectivity requirements [#connectivity-requirements]
+
+As a summary, the integration needs to be configured and allowed to connect to:
+
+    * Hosts listed in zookeeper_hosts over the Zookeeper protocol, using the Zookeeper authentication mechanism (if autodiscover_strategy is set to zookeeper).
+    *Hosts defined in bootstrap_broker_host over the Kafka protocol, using the Kafka broker’s authentication/transport mechanisms (if autodiscover_strategy is set to bootstrap).
+    * All brokers in the cluster over the Kafka protocol and port, using the Kafka brokers' authentication/transport mechanisms.
+    * All brokers in the cluster over the JMX protocol and port, using the authentication/transport mechanisms specified in the JMX configuration of the brokers.
+    * All producers/consumers specified in producers and consumers over the JMX protocol and port, if you want producer/consumer monitoring. JMX settings for the consumer must be the same as for the brokers.
+    
+<Callout variant="important">
+
+For the cloud: By default, Security Groups (and their equivalents in other cloud providers) in AWS do not have the required ports open by default. JMX requires two ports in order to work: the JMX port and the RMI port. These can be set to the same value when configuring the JVM to enable JMX and must be open for the integration to be able to connect to and collect metrics from brokers.
```

Comment:
"For the cloud" and "must" in bold in the original doc.

Reply:
@urbiz-nr, I added the bold elements and also noticed I had missed some code formatting; so, I added that, too. Thanks so much!